cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_info.tsx

```diff
@@ -0,0 +1,42 @@
+//react and css
+import React, { useState, useEffect } from "react";
+import NFTBuyButton from "./nft_buy_button/nft_buy_button";
+import s from "./nft_info.module.scss";
+
+//external exports
+
+//internal imports
+import { nft } from "../../../../lib/data_types";
+
+export default function NFTInfo(props: { nft: nft }) {
+
+    return (
+        <div className={s.container}>
+            <div className={s.left}>
+                <NFTCollection {...props} />
+                <NFTName {...props} />
+            </div>
+            <NFTBuyButton {...props} />
```

Comment:
Don't send all props, if not all props are needed. For NftName, only send name, for NftBuyButton, only send price and externalUrl for example